ci-operator/config/openstack-k8s-operators/infra-operator/openstack-k8s-operators-infra-operator-master.yaml

```diff
@@ -0,0 +1,47 @@
+base_images:
+  golangci-lint:
+    name: golangci-lint
+    namespace: ci
+    tag: v1.46.0
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: unit
+  commands: make gotest
+  container:
+    from: src
+- as: govet
+  commands: make govet
```

Comment:
govet: already in pre-commit, we can remove from here.

Reply:
ack, thanks